### PR TITLE
Fixed support for Binders, Fixed handling of binders in One2Manies Mapping

### PR DESCRIPTION
**Pull request by toggm**
added binder test, fixed type mapping of binders for basic types
fixed parameterbinding of onetomanies mapping (> 2)

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -39,7 +39,7 @@ lazy val baseSettings = Seq(
   incOptions := incOptions.value.withNameHashing(true),
   scalatestVersion := "3.0.3",
   specs2Version := "3.9.4",
-  //scalaVersion := "2.11.11",
+  // scalaVersion := "2.11.11",
```
Comment (seratch) [inline build.sbt]: Diff here is unnecessary. Can you revert this?

File: scalikejdbc-core/src/main/scala/scalikejdbc/WrappedResultSet.scala [hunk 2]
```diff
@@ -31,9 +31,10 @@ case class WrappedResultSet(underlying: ResultSet, cursor: ResultSetCursor, inde
     try {
       op
     } catch {
-      case e: Exception => throw new ResultSetExtractorException(
-        "Failed to retrieve value because " + e.getMessage + ". If you're using SQLInterpolation, you may mistake u.id for u.resultName.id.", Some(e)
-      )
+      case e: Exception =>
```
Comment (seratch) [inline scalikejdbc-core/src/main/scala/scalikejdbc/WrappedResultSet.scala]: This change looks unnecessary this time. Can you revert this, too?

**Comment by toggm** [comment]
Reverted unnecessary changes, used them in my local tests...

**Comment by toggm** [comment]
btw. when I try to integrate your project into my own project using Scala version 2.11.x using a github ProjectRef I receive the following exception:
sbt.ResolveException: unresolved dependency: org.scalikejdbc#scalikejdbc_2.11;3.1.0-SNAPSHOT: configuration not found in org.scalikejdbc#scalikejdbc_2.11;3.1.0-SNAPSHOT: 'master'. It was required from org.scalikejdbc#scalikejdbc-syntax-support-macro_2.11;3.1.0-SNAPSHOT compile

Did you encounter this before? How can I make your master branch "backward" compatible?

**Comment by seratch** [comment]
>Did you encounter this before? How can I make your master branch "backward" compatible?

I myself have never tried using project ref.

**Comment by seratch** [comment]
The changes look ready to merge. Thanks!

**Pull request merged**